R/prettyOncoplot.R

```diff
@@ -8,13 +8,21 @@
 #' sample_id, pathology
 #'
 #' @param maf_df A maf as data frame containing the mutations you want to plot.
-#' @param cnv_df An optional data frame of CN status for genes you want included (rows = sample_id, columns = Hugo_Symbol)
+#' @param gene_cnv_df An optional data frame of CN status for genes you want included (rows = sample_id, columns = Hugo_Symbol)
+#' See [GAMBLR.results::get_cnv_and_ssm_status] or [GAMBLR.open::get_cnv_and_ssm_status] for more information.
```

Comment:
Just flagging this because this reference will conflict for external users. They won't have the .results and there is no get_cnv_and_ssm_status function in the .open